<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tool</class>
 <widget class="QMainWindow" name="Tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASR Post Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_tool">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="playerLayout">
        <item>
         <widget class="VideoWidget" name="m_videoWidget" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QSlider" name="slider_position">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_position">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>00:00 / 00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="PlayerControls" name="m_playerControls" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_editor">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="Editor" name="m_editor"/>
          <widget class="WordEditor" name="m_wordEditor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="TagListDisplayWidget" name="m_tagListDisplay" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="close"/>
   </widget>
   <widget class="QMenu" name="menuMedia_Player">
    <property name="title">
     <string>Player</string>
    </property>
    <addaction name="player_open"/>
    <addaction name="player_togglePlay"/>
    <addaction name="player_seekForward"/>
    <addaction name="player_seekBackward"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="editor_openTranscript"/>
    <addaction name="editor_save"/>
    <addaction name="editor_saveAs"/>
    <addaction name="editor_close"/>
    <addaction name="separator"/>
    <addaction name="editor_debugBlocks"/>
    <addaction name="editor_jumpToLine"/>
    <addaction name="separator"/>
    <addaction name="editor_splitLine"/>
    <addaction name="editor_mergeUp"/>
    <addaction name="editor_mergeDown"/>
    <addaction name="editor_toggleWords"/>
    <addaction name="separator"/>
    <addaction name="editor_changeLang"/>
    <addaction name="editor_changeSpeaker"/>
    <addaction name="editor_propagateTime"/>
    <addaction name="editor_editTags"/>
    <addaction name="separator"/>
    <addaction name="editor_autoSave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="edit_undo"/>
    <addaction name="edit_redo"/>
    <addaction name="separator"/>
    <addaction name="edit_cut"/>
    <addaction name="edit_copy"/>
    <addaction name="edit_paste"/>
    <addaction name="separator"/>
    <addaction name="edit_findReplace"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="view_incFont"/>
    <addaction name="view_decFont"/>
    <addaction name="view_font"/>
    <addaction name="view_toggleTagList"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="help_keyboardShortcuts"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuMedia_Player"/>
   <addaction name="menuEditor"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="close">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="player_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="editor_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="editor_save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="editor_openTranscript">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="editor_debugBlocks">
   <property name="text">
    <string>Debug Blocks</string>
   </property>
  </action>
  <action name="editor_jumpToLine">
   <property name="text">
    <string>Jump to Highlighted Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="player_togglePlay">
   <property name="text">
    <string>Play / Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="editor_splitLine">
   <property name="text">
    <string>Split Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+;</string>
   </property>
  </action>
  <action name="editor_mergeUp">
   <property name="text">
    <string>Merge Up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="editor_mergeDown">
   <property name="text">
    <string>Merge Down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="editor_findReplace">
   <property name="text">
    <string>Find Replace</string>
   </property>
  </action>
  <action name="editor_toggleWords">
   <property name="text">
    <string>Toggle Word Editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="player_seekForward">
   <property name="text">
    <string>Seek Forward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="player_seekBackward">
   <property name="text">
    <string>Seek Backward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="edit_undo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="edit_redo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="edit_cut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="edit_copy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="edit_paste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="edit_findReplace">
   <property name="text">
    <string>Find / Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="view_incFont">
   <property name="text">
    <string>Increase Font Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="view_decFont">
   <property name="text">
    <string>Decrease Font Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="editor_changeSpeaker">
   <property name="text">
    <string>Change Speaker</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="editor_propagateTime">
   <property name="text">
    <string>Propagate Time</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="help_keyboardShortcuts">
   <property name="text">
    <string>Keyboard Shortcuts</string>
   </property>
  </action>
  <action name="view_font">
   <property name="text">
    <string>Change Font</string>
   </property>
  </action>
  <action name="view_toggleTagList">
   <property name="text">
    <string>Toggle TagList</string>
   </property>
  </action>
  <action name="editor_editTags">
   <property name="text">
    <string>Edit Tags</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+'</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>jd</string>
   </property>
  </action>
  <action name="editor_saveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="editor_close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="editor_changeLang">
   <property name="text">
    <string>Change Transcript Language</string>
   </property>
  </action>
  <action name="editor_autoSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>mediaplayer/videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayerControls</class>
   <extends>QWidget</extends>
   <header>mediaplayer/playercontrols.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Editor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor/editor.h</header>
  </customwidget>
  <customwidget>
   <class>TagListDisplayWidget</class>
   <extends>QWidget</extends>
   <header>editor/utilities/taglistdisplaywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WordEditor</class>
   <extends>QTableWidget</extends>
   <header>editor/wordeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
